### PR TITLE
[FVT] Update runcmdinstaller test case  for timing issue.

### PR DESCRIPTION
**Pull request by caomengmeng**
@immarvin @tingtli 
Update runcmdinstaller test case for the timing issue.
Please help to review code.
Thanks.

### CODE REVIEW

File: xCAT-test/autotest/testcase/runcmdinstaller/cases0 [hunk 1]
```diff
@@ -12,9 +12,8 @@ cmd:nodeset $$CN osimage=__GETNODEATTR($$CN,os)__-__GETNODEATTR($$CN,arch)__-ins
 check:rc==0
 cmd:if [[ "__GETNODEATTR($$CN,arch)__" =~ "ppc64" ]]; then rnetboot $$CN;elif [[ "__GETNODEATTR($$CN,arch)__" =~ "x86_64" ]];then rpower $$CN boot; fi
 check:rc==0
-cmd:sleep 300
-cmd:xdsh $$CN date
-check:output=~$$DATE
+cmd:a=0;while ! `lsdef -l $$CN|grep status|grep installing >/dev/null`; do sleep 20;((a++));if [ $a -gt 30 ];then break;fi done
```
Comment (immarvin) [inline xCAT-test/autotest/testcase/runcmdinstaller/cases0]: ``lsdef -l $$CN|grep status`` can be replaced with ``lsdef $$CN -i status``

Comment (immarvin) [inline xCAT-test/autotest/testcase/runcmdinstaller/cases0]: if the loop counter exceeds 30, will the "runcmdinstaller" test below run?

Comment (caomengmeng) [inline xCAT-test/autotest/testcase/runcmdinstaller/cases0]: @immarvin   below is the results with the two commands, and for FVT, the ```lsdef $$CN | grep status``` is more helpful to debug with the ```statustime``` attribute value to make the installation time clearly, too.
```
[root@c910f02c04p08 xcatdebugmode]# lsdef c910f02c04p09 | grep status
    status=booted
    statustime=06-21-2016 02:23:23
    updatestatus=synced
    updatestatustime=04-12-2016 04:39:09
[root@c910f02c04p08 xcatdebugmode]# lsdef c910f02c04p09 -i status
Object name: c910f02c04p09
    status=booted
```

Comment (caomengmeng) [inline xCAT-test/autotest/testcase/runcmdinstaller/cases0]: @immarvin  Yes, if the looper exceeds 30, the following test scripts will be executed also, but they will failed.

**Pull request merged**